app/assemblers/assemble_exercise_representations_with_feedback.rb

```diff
@@ -22,7 +22,7 @@ def representations
       mentor:,
       track:,
       status: :with_feedback,
-      criteria: params[:criteria],
+      criteria: params.fetch(:criteria, ''),
```

Comment:
Why is this needed?

Reply:
I don't understand 🙂 

This is passing it into a search method, whereas #2872 is changing what is rendered out (in the `query:` line). Unless I'm missing something, this won't change what React sees and is effectively a noop.

Reply:
Ehm, feck me. You're right 🤦